### PR TITLE
build(deps): Upgrade to Deckgl 9.0.12

### PR DESCRIPTION
**Pull request by hkfb**

### CODE REVIEW

**Review by nilscb — APPROVED** [review]

**Review by LeonidPolukhin — APPROVED** [review]

File: typescript/packages/subsurface-viewer/src/layers/map/fragment.fs.glsl.ts [hunk 1]
```diff
@@ -31,27 +32,22 @@ uniform bool isClampColor;
...
-   normal[2] /= 127.0;
 
    if (!smoothShading || (normal[0] == 0.0 && normal[1] == 0.0 && normal[2] == 0.0)) {
       normal = normalize(cross(dFdx(position_commonspace.xyz), dFdy(position_commonspace.xyz)));
    }
 
    //Picking pass.
-   if (picking_uActive && !picking_uAttribute) {
-      gl_FragColor = encodeVertexIndexToRGB(vertexIndex);
+   if (picking.isActive > 0.5 && !(picking.isAttribute > 0.5)) {
```
[the remark this answers is not in the document]
Comment (nilscb) [reply typescript/packages/subsurface-viewer/src/layers/map/fragment.fs.glsl.ts:45]: They are floats now. Used to be booleans.

**Pull request merged**